src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -18,8 +19,15 @@ public SelectTestFigureOptionListener(DriverManager driverManager) {
 	public void actionPerformed(ActionEvent e) {
 		if (e.getActionCommand().toLowerCase().equals("figure joe 1")) {
 			FiguresJoe.figureScript1(driverManager.getCurrentDriver());
-		} else if (e.getActionCommand().toLowerCase().equals("figure joe 2")) {
+		}
+		else if (e.getActionCommand().toLowerCase().equals("figure joe 2")) {
 			FiguresJoe.figureScript2(driverManager.getCurrentDriver());
 		}
+		else if(e.getActionCommand().toLowerCase().equals("figure command")){
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.